### PR TITLE
Cart

### LINKED ISSUE #5: Add Cart Page

### PR DESCRIPTION
**Pull request by ZuberDunge**
image

### CODE REVIEW

File: docs/login.html [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" href="footer.css" />
+    <link rel="stylesheet" href="category.css" />
+    <link rel="stylesheet" href="nav.css" />
+    <link rel="stylesheet" href="login-signup.css" />
+    <style>
```
Comment (saadahmadhere) [inline docs/login.html]: you forgot to remove the internal stylesheet.

File: docs/login.html [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+            </div>
+            <span class="material-icons"> favorite </span>
+          </div>
+        </nav>
+      </div>
+
+      <div class="hero-img-cat login-section-wala">
+        <img
+          src="./images/Batman-Comics-10-Essentials-You-Need-To-Read.jpg"
+          alt=""
```
Comment (saadahmadhere) [inline docs/login.html]: alt attribute is empty

File: docs/login.html [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+        />
+        <div class="absolute-head">Welcome to BatStore</div>
+      </div>
+
+      <div class="login-section">
+        <div class="login-img">
+          <img
+            src="https://www.posters.eu/media/catalog/product/cache/cb3faf85ecb1e071fdba48f981c86454/g/b/gb_mp1588-dc-comics-cast-5028486240852.jpg"
+            alt=""
```
Comment (saadahmadhere) [inline docs/login.html]: here too.

File: docs/login.html [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+          />
+        </div>
+        <div
+          class="login-form bat-flex bat-flx-dir-col bat-align-center bat-justify-center-center flex-gap-1"
+        >
+          <h2>Hello there!</h2>
+          <form action="">
```
Comment (saadahmadhere) [inline docs/login.html]: action is empty

File: docs/login.html [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+            <ul class="bat-footer-nav">
+              <li class="bold-600 bat-fs-14px">Help</li>
+              <li>Return</li>
+              <li>Payment</li>
+              <li>Term of use</li>
+              <li>Privacy</li>
+              <li>Payment options Available</li>
+            </ul>
+
+            <!-- <div>
```
Comment (saadahmadhere) [inline docs/login.html]: comment should be removed here

File: style/login-signup.css [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+    transition: all 0.2s ease-in-out;
+}
+
+.input-parent input:valid+label,
+.input-parent input:focus+label {
+    font-size: 14px;
+    transform: translateY(0);
+}
+
+@media screen and (max-width: 750px) {
```
Comment (saadahmadhere) [inline style/login-signup.css]: relative units would be more appreciated. Just a suggestion

**Review by saadahmadhere — CHANGES_REQUESTED** [review]
Great work! Nice and clean code and file structure. But there are still some cleanup required as mentioned in the comments.

File: docs/cart.html [hunk 7]
```diff
@@ -0,0 +1,462 @@
...
+                <div class="bat-flex bat-align-center bat-fs-1r pointer">
+                  <span class="material-icons"> shopping_cart </span>
+                  <span class="hide-on-phone"> Cart </span>
+                </div>
+                <span class="material-icons"> favorite </span>
+              </div>
+            </nav>
+          </div>
+
+
```
Comment (saadahmadhere) [inline docs/cart.html:52]: extra space should be avoided

File: docs/cart.html [hunk 8]
```diff
@@ -0,0 +1,462 @@
...
+                                <div class="bat-flex bat-gap-1 bat-align-center m-tb-8">
+                                    <div class="rating">
+                                        4 <span class="material-icons"> star </span>
+                                    </div>
+                                    <div class="bat-fs-12px">
+                                        Free Delivery
+                                    </div>
+                                </div>
+
+
```
Comment (saadahmadhere) [inline docs/cart.html:105]: here too

File: docs/cart.html [hunk 9]
```diff
@@ -0,0 +1,462 @@
...
+
+
+
+
+
+
+
+        <!-- <div class="main featured-area-flex bat-flex bat-align-center flex-gap-1">
```
Comment (saadahmadhere) [inline docs/cart.html:335]: unwanted comments and space should be removed

File: docs/cart.html [hunk 10]
```diff
@@ -0,0 +1,462 @@
...
+                    <li class="bold-600 bat-fs-14px">Help</li>
+                    <li>Return</li>
+                    <li>Payment</li>
+                    <li>Term of use</li>
+                    <li>Privacy</li>
+                    <li>Payment options Available</li>
+                </ul>
+
+                <!-- <div>
+              <img
```
Comment (saadahmadhere) [inline docs/cart.html:443]: here too

File: style/cart.css [hunk 11]
```diff
@@ -0,0 +1,132 @@
+.cart {
+    max-width: 1200px;
```
Comment (saadahmadhere) [inline style/cart.css:2]: using relative units is a good practice

**Review by saadahmadhere — CHANGES_REQUESTED** [review]
Well written code. Again, some cleanups and refactoring required as requested.

File: docs/cart.html [hunk 12]
```diff
@@ -0,0 +1,462 @@
...
+    <link rel="stylesheet" href="footer.css" />
+    <link rel="stylesheet" href="category.css" />
+    <link rel="stylesheet" href="nav.css" />
+</head>
+
+<body>
+    <div class="app">
+        <div class="navbar">
+            <nav>
+              <div class="bat-flex media-nav bat-align-center flex-gap-1">
```
Comment (Shivanshsharma01) [inline docs/cart.html:21]: It's good to wrap the main nav in a "header" tag

File: docs/signup.html [hunk 13]
```diff
@@ -0,0 +1,255 @@
...
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="index.css" />
+    <link rel="stylesheet" href="footer.css" />
+    <link rel="stylesheet" href="category.css" />
+    <link rel="stylesheet" href="nav.css" />
+    <link rel="stylesheet" href="login-signup.css" />
+    <style>
+      .slider-box {
```
Comment (Shivanshsharma01) [inline docs/signup.html]: better to remove internal style sheet from here too

File: style/cart.css [hunk 14]
```diff
@@ -0,0 +1,132 @@
...
+}
+
+.cart-grid {
+    display: grid;
+    grid-template-columns: 2fr 1fr;
+}
+
+.horizointal-card.bat-card img {
+    width: 120px;
+    height: 130px;
```
Comment (Shivanshsharma01) [inline style/cart.css:13]: can use rem here exact values are not required

**Review by Shivanshsharma01 — COMMENTED** [review]
The code is clean with good use of variables. Can use relative units above 16px, try implementing both reviewer's suggestions

File: docs/login.html (hunk 1 above)
Comment (ZuberDunge) [reply docs/login.html]: Thanks for noticing Saad, I've updated it in Login PR

File: docs/login.html (hunk 2 above)
Comment (ZuberDunge) [reply docs/login.html]: Thank for noticing, Fixed.

File: docs/login.html (hunk 5 above)
Comment (ZuberDunge) [reply docs/login.html]: Done

File: docs/cart.html (hunk 7 above)
Comment (ZuberDunge) [reply docs/cart.html:52]: Will update it

File: docs/cart.html (hunk 9 above)
Comment (ZuberDunge) [reply docs/cart.html:335]: DONE

File: docs/signup.html (hunk 13 above)
Comment (ZuberDunge) [reply docs/signup.html]: yes, yes i've updated it

File: style/cart.css (hunk 14 above)
Comment (ZuberDunge) [reply style/cart.css:13]: I'll fix it

**Pull request merged**